src/ui/input/color-picker/hidden-color-input.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { PureComponent, ChangeEvent } from 'react';
+import { Input } from '@teambit/base-ui.input.checkbox.label';
+
+export type HiddenColorInputProps = React.InputHTMLAttributes<HTMLInputElement>;
+
+type HiddenColorInputState = {
+  isOpen: boolean;
+  type: string;
+};
+
+export class HiddenColorInput extends PureComponent<HiddenColorInputProps, HiddenColorInputState> {
+  state = {
+    isOpen: false,
+    type: 'color',
+  };
+
+  handleOnChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const { onChange } = this.props;
+    onChange && onChange(e);
+  };
```

Comment:
this is no op, remove

Reply:
I replaced this:
```
handleOnChange = (e: ChangeEvent<HTMLInputElement>) => {
    const { onChange } = this.props;
    onChange && onChange(e);
  };
```

With this, call directly the function from the props:
```
<Input
        {...this.props}
        type={type}
        onChange={this.props.onChange}
        onClick={this.handleOnClick}
        onBlur={this.handleOnClick}
      />
```